mount/mount_unix.go

```diff
@@ -59,3 +63,62 @@ func UnmountRecursive(target string, flags int) error {
 	}
 	return nil
 }
+
+func unmount(target string, flags int) error {
+	if isFUSE(target) {
+		if err := unmountFUSE(target); err == nil {
+			return nil
+		}
```

Comment:
If it is FUSE and we failed, we just continue on down into the regular umount path. Should we just `return unmountFUSE(target)` here instead?

Reply:
No idea, this code was writter this way before me. I just moved it to a different file.

Reply:
Oh I'm realizing now it wasn't altered.. interesting

Reply:
It would make more sense if the code looked like

```go
if isFuse(dir) {
  return umountFuse(dir)
```

But I have no idea if that will not break anything.

Reply:
Agree, regular unmount afterwards seems mostly a fallback. Can be an exploration for later